### ISSUE TITLE
Batch engine dependabot updates into a single commit

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#134729]
It's not an efficient use of CI resources to update the version of each package individually in separate commits. This should behave like the framework package autoroller, and update all packages at once ~1 time a day. Looks like this was just added in https://github.com/flutter/engine/pull/45754. cc @godofredoc 

<img width="997" alt="Screenshot 2023-09-14 at 6 26 42 AM" src="https://github.com/flutter/flutter/assets/6343103/1f514e17-718c-4012-ae88-29aa92e35c74">

**Comment by zanderso** [comment]
Ideally `web_ui` would use the DEPS file for all its dependencies so that we can avoid having two dependency management systems in the engine repo. cc @yjbanov

**Comment by zanderso** [comment]
For example `package:archive` is already pulled by the DEPS file: https://github.com/flutter/engine/blob/main/DEPS#L643, so we have two different dependency management systems pulling two versions of the same package. Instead of investing in having dependabot update these dependencies, we should use autorollers that update the DEPS file as is used for all the other engine dependencies.

**Comment by yjbanov** [comment]
I agree that we should move to the DEPS file. Please file an issue for team-web, platform-web.

**Comment by zanderso** [comment]
I filed https://github.com/flutter/flutter/issues/144024

**Comment by matanlurey** [comment]
Does team-web want to own this? Otherwise close :)
